source/includes/table-beta-compatibility.rst

```diff
@@ -50,11 +56,29 @@
      - |checkmark|
      - 
      - |checkmark|
+     - |checkmark|
 
    * - Destination Data Handling 
      - |checkmark|
      - |checkmark|
      - |checkmark|
      - |checkmark|
      - |checkmark|
-     - 
\ No newline at end of file
+     - 
+     - |checkmark|
+
+   * - Pre-6.0 Version Support
+     - |checkmark|
+     - |checkmark| [#pre-6.0-abc]_
+     - 
+     -
+     - |checkmark|
+     - |checkmark|
+     -
+
+.. [#pre-6.0-abc]
+
+   A->B->C migrations are compatible with pre-6.0 version support only if 
+   cluster A uses a server version older than 6.0. If cluster A and B are both 
+   pre-6.0 clusters, then the second migration (B->C) can start only after all
+   ``mongosync``-generated :ref:`DDLs <txn-prod-considerations-ddl>` complete.
```

Comment:
@ajayvijayakumar123 I had added this note to the docs ticket based on internal discussion - is there anything an external user can do to ensure the mongosync-generated DDLs are complete? If not, we can just state that A can only be pre-6.0.

Reply:
Maria and I spoke on slack about this. I think we should say that if B is pre-6.0, B-->C can only start if A-->B committed first. That's the safest way to say that all mongosync-generated DDLs are complete from A-->B.

(I also updated the DOCSP ticket to reflect this)

Reply:
Tweaked the wording a bit but updated!